src/main/java/bio/terra/service/auth/iam/sam/SamIam.java

```diff
@@ -357,6 +356,12 @@ private void createProfileResourceInnerV2(AuthenticatedUserRequest userReq, Stri
     samResourceApi.createResourceV2(IamResourceType.SPEND_PROFILE.toString(), req);
   }
 
+  private @NotNull List<String> getAdminEmailList() {
```

Comment:
This seems like a halfway refactor, I would instead follow @okotsopoulos 's suggestion and move the `putPoliciesItem()` in here too.

Reply:
The reason I didn't is because these are different policies on different kinds of resources. They just happen to be named the same. Bundling them together in a single function artificially couples them and barely saves anything.